Branch-SDK/Branch-SDK/BranchUniversalObject.m

```diff
@@ -0,0 +1,137 @@
+//
+//  BranchUniversalObject.m
+//  Branch-TestBed
+//
+//  Created by Derrick Staten on 10/16/15.
+//  Copyright © 2015 Branch Metrics. All rights reserved.
+//
+
+#import "BranchUniversalObject.h"
+#import "BNCError.h"
+#import "BranchConstants.h"
+
+@implementation BranchUniversalObject
+
+- (instancetype)initWithCanonicalIdentifier:(NSString *)canonicalIdentifier {
+    if (self = [super init]) {
+        self.canonicalIdentifier = canonicalIdentifier;
+    }
+    return self;
+}
+
+- (instancetype)initWithTitle:(NSString *)title {
+    if (self = [super init]) {
+        self.title = title;
+    }
+    return self;
+}
+
+- (void)registerView {
+    Branch *branch = [Branch getCurrentInstanceIfAny];
+    if (!branch) {
```

Comment:
This faulty logic can also be found in the share sheet. @aaustin I could emulate the existing faulty logic and simply call `[Branch getInstance]`, remove the new method, and remove the checks for whether it's been invoked or not